lib/driver.js

```diff
@@ -399,15 +399,12 @@ class AndroidUiautomator2Driver extends BaseDriver {
   async initAUT () {
     // Uninstall any uninstallOtherPackages which were specified in caps
     if (this.opts.uninstallOtherPackages) {
-      let uninstallOtherPackages;
-      try {
-        uninstallOtherPackages = helpers.parseArray(this.opts.uninstallOtherPackages);
-      } catch (e) {
-        logger.errorAndThrow(`Could not parse "uninstallOtherPackages" capability: ${e.message}`);
-      }
-      await helpers.uninstallOtherPackages(uninstallOtherPackages, this.adb, [
-        SETTINGS_HELPER_PKG_ID, SERVER_PACKAGE_ID, SERVER_TEST_PACKAGE_ID
-      ]);
+      helpers.validateDesiredCaps(this.opts);
```

Comment:
This is already done: https://github.com/appium/appium-uiautomator2-driver/blob/master/lib/driver.js#L143-L145